sounds/models.py

```diff
@@ -124,6 +124,53 @@ def random(self, excludes=None):
         else:
             return None
 
+    def bulk_query_solr(self, where, order_by, limit, args):
+        query = """SELECT
```

Comment:
Please add documentation here to explain why we added this method, which is a version of `bulk_query` which selects all relevant information for indexing sounds to solr.